Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Media-Picker-3/index.md

```diff
@@ -0,0 +1,155 @@
+---
+versionFrom: 8.13.0
+---
+
+# Media Picker #
+
+`Alias: Umbraco.MediaPicker3`
+
+`Returns: IEnumerable<MediaWithCrops>` or `MediaWithCrops`
+
+This property editors returns a single `MediaWithCrops` item if the "Pick multiple items" data type setting is disabled or a collection if it is enabled.
+
+## Data Type Definition Example
+
+![Media Picker Data Type Definition](images/Media-Picker3-DataType.jpg)
+
+### Accepted types
+
+Use setting to limit the picker to only select Media Items of these types.
+
+### Pick multiple items
+
+Use this setting to enable the property to contain multiple items, when this is turned on the property editor returns a `IEnumerable<MediaWithCrops>`. You can still set the Maximum Amount to 1, do so if you like to retrive a collection but only allowing the Content Editors to select one Media Item.
+
+### Amount
+
+Use this setting to enforce a certain minimum or maximum amount of selected Media Items, the maximum setting is unuseable when multiple items is disabled.
+
+### Start node
+
+Limit the picker to only work in a certain part of the Media Tree.
+
+### Ignorer user start nodes
+
+Use setting to overrule user permissions, to enable any user of this property to pick any Media Item of the choosen Start node.
+
+When this setting is enabled, a user who doesn't normally have access to the media selected as "Start Node" (/Design in this case), can access the media when using this particular Media Picker. If no Start node has been defined for this property any content can be viewed and selected of this property.
+
+
+### Image Croppings
+
+Define local image croppings, this data will stay locally, meaning that this will be stored on the document in this property.
```

Comment:
I'm having a hard time understanding what is meant by this sentence.

Would you be able to help me understand?

Reply:
The existing Image Cropper feature is stored together with the Image-file in the Image Cropper Property Editor. Default is that its used on the Media Type 'Image'. When setting Image Crops on the Image Cropper, they are then shared by all usages of that Media Item. aka. Crops made on Photo of Umbraco HQ, will be the same everywhere that Media Item is used.

The Local Image Crops Feature of MediaPicker 3, is a feature where the crops are stored together with the pick, not on the Media Item itself. Enabling differentiation, as each media item will have custom crops for each usage.

I did reformulate a bit, but please help me out, its rather complex to get everyone's understanding right :-)

Reply:
Ah! Alright. I understand it now. I like the new wording you've added 💪 Let's stick with that.